samcli/commands/build/command.py

```diff
@@ -101,6 +107,23 @@
     help="If your functions depend on packages that have natively compiled dependencies, use this flag "
     "to build your function inside an AWS Lambda-like Docker container",
 )
+@click.option(
+    "--container-env-vars",
```

Comment:
Shouldn't this be `--container-env-var`?

Reply:
This is designed this way to be consistent with the image-repositories tag in deploy command. This has been finalized with Mark and UX team.

Reply:
Naming change confirmed. (Just to record this in conversation, I know you are aware already)